creme/geolocation/tests/test_bricks.py

```diff
@@ -418,3 +419,33 @@ def test_osm_neighbours(self):
             f"""tileMapAttribution: '&copy; <a href="{cright_url}">{cright_title}</a>'""",
             script_node.text,
         )
+
+    @skipIfCustomAddress
+    @parameterized.expand([
+        OpenStreetMapNeighboursMapBrick,
+        OpenStreetMapDetailMapBrick,
+        GoogleNeighboursMapBrick,
+        GoogleDetailMapBrick,
+    ])
+    def test_disabled_bricks(self, brick_class):
+        # user = self._simple_login()
+        user = self.login_as_root_and_get()
+
+        contact = Contact.objects.create(last_name='Contact 1', user=user)
+
+        BrickDetailviewLocation.objects.create_if_needed(
+            brick=brick_class,
+            model=type(contact),
+            order=50,
+            zone=BrickDetailviewLocation.BOTTOM,
+        )
+
+        response = self.assertGET200(contact.get_absolute_url())
+        self.assertContains(response, _("No address defined for now"))
+
+        contact.trash()
+
+        response = self.assertGET200(contact.get_absolute_url())
+        self.assertContains(
+            response, _("The geolocation feature is disabled for entities in the trash")
+        )
```

Comment:
Use 'BrickTestCaseMixin.get_brick_node()' to be sure that the "empty" message is found in the right brick (it could be present several times in the same document -- because there are several geolocation bricks by default on detail-view).